core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -902,6 +915,16 @@ private ExprValue exprMinute(ExprValue time) {
     return new ExprIntegerValue(time.timeValue().getMinute());
   }
 
+  /**
+   * Minute_of_day implementation for ExprValue.
+   *
+   * @param time ExprValue of Time/String type.
+   * @return ExprValue.
+   */
+  private ExprValue exprMinuteOfDay(ExprValue time) {
+    return new ExprIntegerValue(time.timeValue().getHour()*60 + time.timeValue().getMinute());
```

Comment:
Maybe replace `60` with `private static final int MINUTES_PER_HOUR = 60;` so we don't have magic numbers.

Reply:
* as in, make that declaration at start of class so it can be shared with other DateTime functions.

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/189/commits/6bfe0c66f307ffe281e7eb0675a081577874cbed